.eslintrc.json

```diff
@@ -1,8 +1,10 @@
 {
+  "root": true,
   "extends": [
     "./commonjs/.eslintrc.json",
     "./browser/.eslintrc.json",
     "./es6/.eslintrc.json",
-    "./react/.eslintrc.json"
+    "./react/.eslintrc.json",
+    "./typescript/.eslintrc.json"
```

Comment:
Should we extend the typescript config by default? Might be neat to leave it out so that projects that still doesn't use typescript won't load it. Don't know if it really changes any behaviour though so might be unnecessary

Reply:
Good point, I will take a look!